sdk/iot/ci.yml

```diff
@@ -30,5 +30,7 @@ extends:
   parameters:
     ServiceDirectory: iot
```

Comment:
I'm not sure the models repository assets should be rooted in `/sdk/iot/`. In both dotnet and java its rooted in `/sdk/modelsrepository/` and this is what was approved by the architecture board.

Reply:
We all approved this namespace. If you want to change it now, this is a different discussion to be had a later date.

Reply:
I'm not referring to the namespace, I am referring to where in the sdk mono repo this SDK should exist.

According to the [Azure SDK repo directory structure](https://github.com/Azure/azure-sdk/blob/master/docs/policies/repostructure.md#sdk-directory-layout) the pattern is `sdk/<service name>/<package name>/*` where `<service name>` - should be the short name for the azure service.

This is the models repository service and therefore `sdk/modelsrepository/*` as the service root makes sense. And dotnet and java share `/sdk/modelsrepository/*`

Reply:
As this is a change with both infrastructure and semantic implications, I would much prefer it be done in a separate PR after a discussion involving all relevant parties.

Reply:
Ok. We could punt to a subsequent PR.

But note the repo directory structure spec for the `<service>` part also states "These names should match across all the different language repos..." and we've already committed to `modelsrepository` for 2 other languages.

@johanste what do you think about refactoring placement of the Python models repository SDK in `sdk/modelsrepository/` as its committed in dotnet and Java, versus the current placement in `sdk/iot/`?

Reply:
@annatisch , @lmazuel for more perspectives.

Reply:
FWIW I would say that I do completely agree that all languages should eventually be organized the same way. The question is which is most correct? - the majority of clients are organized according to their <group> namespace. There are exceptions here and there, but I don't know what goes into that decision exactly, and if we don't consider this part of the `iot` folder, I would have to wonder why we're in the `iot` namespace at all.

Reply:
According to the Azure SDK guidelines, the root folder name should be the name of the service.

From Laurent Mazuel:
    
> <https://github.com/Azure/azure-sdk/blob/master/docs/policies/repostructure.md#sdk-directory-layout>
> This is the Azure SDK parent repository and mostly contains documentation around guidelines and policies as well as the releases for the various languages supported by the Azure SDK.
> The folder name should be the service name, not the group name
​> So according to our own guidelines, "deviceupdate" is the right folder name to me